package.json

```diff
@@ -28,7 +28,7 @@
     "backbone": "1.2.3",
     "backbone-poller": "^1.1.3",
     "browserify-shim": "3.8.12",
-    "camshaft-reference": "0.4.0",
+    "camshaft-reference": "1.x || <1.x",
```

Comment:
What does this mean? Do we want to enforce 1.x? Why do we need the `|| <1.x` part?